docs/manual/cluster-admin/alerting-and-troubleshooting.md

```diff
@@ -0,0 +1,130 @@
+
+Solutions:
+
+  1. Force remove unhealth containers with this command in terminal:
+  `kubectl delete pod pod-name --grace-period=0 --force`
+  2. Recreate pod in Kubernetes, this operation may block indefinitely because dockerd may not functioning correctly after OOM. If recreate blocked too long, you can log into the node and restart dockerd via `/etc/init.d/docker restart`.
+
+### NodeNotReady Alert
+
+This is a kind of alert from alert manager, and is reported by watchdog service. Watchdog gets such metrics from Kubernetes API. Example metrics is like:
+
+```
+pai_node_count{disk_pressure="false",instance="10.0.0.1:9101",job="pai_serivce_exporter",memory_pressure="false",name="10.0.0.2",out_of_disk="false",pai_service_name="watchdog",ready="true",scraped_from="watchdog-5ddd945975-kwhpr"}
+```
+
+The name label indicate what node this metric represents.
+
+If the node's ready label has value "unknown", this means the node may disconnect from Kubernetes master, this may due to several reasons:
+
+  - Node is down
+  - Kubelet is down
+  - Network partition between node and Kubernetes master
+
+You can first try to log into the node. If you can not, and have no ping response, the node may be down, and you should boot it up.
+
+If you can log in to the node, you should check if the kubelet is up and running, execute `sudo docker ps` command in the node, normally the kubelet container will be listed. The output should like:
+
+```
+  a66a171434cc  gcr.io/google_containers/hyperkube:v1.9.9   "/hyperkube kubele..."   2 weeks ago      Up 2 weeks     kubelet
+```
+
+After this step, you should check the log of kubelet, to see if it can access Kubernetes API. If you see something like:
+
+```
+  E0410 04:24:30.663050    2491 kubelet_node_status.go:386] Error updating node status, will retry: error getting node "10.0.0.1": Get http://10.0.1.2:8080/api/v1/nodes/10.0.0.1: net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)
+```
+
+This means the node can not report its status to Kubernetes, and hence the Kubernetes will post unknown status, and this triggered the alert.
+
+You should check what caused this connectivity problem.
+
+### NodeFilesystemUsage Alert
+
+This is a kind of alert from alert manager, and is used to monitor disk space of each server. If usage of disk space is greater than 80%, this alert will be triggered. OpenPAI has two services may use a lot of disk space. They are storage manager and docker image cache. If there is other usage of OpenPAI servers, they should be checked to avoid the disk usage is caused by outside of OpenPAI.
+
+Solutions:
+
+  1. Check user file on the NFS storage server launched by storage manager. If you didn't set up a storage manager, ignore this step.
+  2. Check the docker cache. The docker may use too many disk space for caching, it's worth to have a check.
```

Comment:
Also should check PAI log folder size. The path is `/var/log/pai`

Reply:
Added the following:

```
3. Check PAI log folder size. The path is `/var/log/pai`.
```